docs/contribution_guide/contributing_code.rst

```diff
@@ -0,0 +1,7 @@
+Contributing Code
+-----------------
+
+.. toctree::
+   :maxdepth: 3
+
+   api_conventions
```

Comment:
I don't think we need this level of hierarchy--talking about the node "api_conventions". For now I'd suggest pulling all of the "api_conventions" content into "contributing_code" (and remove the node api_conventions).